build.py

```diff
@@ -23,18 +27,263 @@ def wait_until_container_is_ready():
     if(p2.match(matched_line)):
         return True
     return False
-    
-def build_and_deploy():
-    subprocess.run(['docker','build','-f','Dockerfile.x86','-t','docker.io/robertbartlettbaron/acct-mgt.x86','.'])
-    subprocess.run(['docker','push','docker.io/robertbartlettbaron/acct-mgt.x86'])
-    subprocess.run(['oc','-n','acct-mgt','rollout','latest','acct-mgt'])
-    result=subprocess.run(['oc','get','all'],stdout=subprocess.PIPE,stderr=subprocess.STDOUT) 
-    return wait_until_container_is_ready()
+
+def build_and_deploy(docker_image):
+    subprocess.run(['docker','build','-f','Dockerfile.x86','-t',docker_image,'.'])
+    subprocess.run(['docker','push',docker_image])
+    subprocess.run(['oc','-n','acct-mgt-2','rollout','latest','acct-mgt-2'])
+    result=subprocess.run(['oc','get','all'],stdout=subprocess.PIPE,stderr=subprocess.STDOUT)
+    return True
```

Comment:
why is this returning `True`? what's the point of having `result` here?

Reply:
All three comments that you pointed out is rob's part of code.But what I understand is, result should be evaluated before returning true or false. And I will make change to this accordingly.

Reply:
Functions shouldn't return anything if you're not using the return value (which you're not).

Reply:
I eventually want to add a check to confirm that the rollout actually happened.  Currently I am not checking if that rollout succeeds or fails, so I am assuming that it is succeeding.